js/catchTheFlow.js

```diff
@@ -115,9 +115,9 @@ function startingGame()
 function update()
 {
 	updateTimer();
-        
-        game.input.enabled=true; 
-   
+
```

Comment:
Those are just  lines with empty spaces that Atom itself has removed. Check that the code changes that has been made by me has been highlighted with a different more bright red/green color.